app/views/layouts/application.html.erb

```diff
@@ -11,18 +11,18 @@
   <body>
   	<div id="common-header">
   		<div class='logo'>
-  			<img width="45" src="https://image.freepik.com/free-icon/reddit-logo_318-67806.jpg">
+  			<img width="36%" src="http://www.iconarchive.com/download/i52596/giannis-zographos/english-football-club/Manchester-United.ico">
   		</div>
```

Comment:
**Suggestion:** It's recommended to use image_tag helper instead of the img tag for better asset pipeline integration. This will also automatically add alt text for the image, improving accessibility.
```suggestion
    <div class='logo'>
    	<%= image_tag('http://www.iconarchive.com/download/i52596/giannis-zographos/english-football-club/Manchester-United.ico', width: '36%') %>
    </div>
```